src/pages/register-a-beacon/more-details.tsx

```diff
@@ -75,9 +76,8 @@ const MoreDetails: FunctionComponent<MoreDetailsPageProps> = ({
       showCookieBanner={showCookieBanner}
       formErrors={form.errorSummary}
       pageText={pageText}
-      id="moreDetails"
-      headingType="label"
     >
+      <FormLabelHeading pageHeading={pageHeading} id="moreDetails" />
```

Comment:
We should update the text area component to use the `hintText` input over the `label` as the label is set through this heading for the input.

See example here: https://design-system.service.gov.uk/get-started/labels-legends-headings/

Reply:
so the `TextareaCharacterCount` component below (which contains the `Textarea` component) has the option to add a label and hint component already, so does hintText need to be added to the `Textarea` component still?

Reply:
changes made